File: java/URI_1014.java
package URI;
import java.util.Locale;
import java.util.Scanner;

public class URI_1014 {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int x = sc.nextInt();
		double y = sc.nextDouble();
		
		double consumoMedio = x / y;
		
		System.out.printf("%.3f km/l%n", consumoMedio);
		
		sc.close();

	}

}

File: kotlin/URI_1014.kt
import java.util.*

fun main(args: Array<String>) {
    Locale.setDefault(Locale.US)
    val sc = Scanner(System.`in`)
    val distancia = sc.nextInt()
    val consumoTotal = sc.nextDouble()
    println("${"%.3f".format(distancia/consumoTotal)} km/l")
}